src/main/java/core/basesyntax/strategy/impl/StrategyOperationImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.StrategyOperation;
+
+public class StrategyOperationImpl implements StrategyOperation {
```

Comment:
Separate Strategy implementations from Operation implementations
They should implement different interfaces